Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -286,9 +311,31 @@ struct RGroupDecompData {
     }
   }
 
+  bool replaceHydrogenCoreDummy(const RGroupMatch &match, RWMol &core,
+                                const Atom &atom, const int currentLabel,
+                                const int rLabel) {
+    // if the R group is just a hydrogen then the attachment point should
+    // replace an existing hydrogen neighbor since all hydrogen neighbors
+    // are copied from the input molecule to the extracted core.
+    if (const auto group = match.rgroups.find(currentLabel);
+        group != match.rgroups.end()) {
```

Comment:
@bp-kelley already commented on it, but this may well become one of my favorite C++17-isms.

Reply:
> @bp-kelley already commented on it, but this may well become one of my favorite C++17-isms.

It's the best new feature I've found